lib/indexers/projects-content/index.js

```diff
@@ -37,9 +80,15 @@ const indexProject = (esClient, project, ProjectVersion) => {
           licenceNumber: project.licenceNumber ? project.licenceNumber.toUpperCase() : null,
           licenceHolder: pick(project.licenceHolder, 'id', 'firstName', 'lastName'),
           establishment: pick(project.establishment, 'id', 'name'),
+          species: extractSpecies(data, project),
+          purposes: extractPurposes(data),
           versionId: version.id,
+          requiresRa: !!project.raDate,
+          continuation: !!version.continuation || !!version['transfer-expiring'],
           protocols,
-          content
+          content: extractContent(data, project),
+          grantedContent: extractContent(data, project, { onlyGranted: true }),
+          ntsContent: extractContent(data, project, { sections: ['aims', 'benefits', 'project-harms', 'fate-of-animals', 'replacement', 'reduction', 'refinement'] })
```

Comment:
What do the structures inside `content`, `grantedContent` and `ntsContent` end up looking like now?

I'm a little concerned that we run the risk of getting close to the field limits if we have the same structure being duplicated through `content` and `grantedContent`

Reply:
content is as before, grantedContent is all fields from "granted" sections, nts is nts section fields. Do you have a different suggestion?

Reply:
So given that the NTS stuff at least is a pure subset of the content, then I'd be tempted to do the filtering of matching sections at search time rather than duplicating it in the index.

i.e search the explicit `[content.aims, content.benefits, etc...]` rather than `content.*`

The granted one is trickier since it's dependent on the schema version, so that may just have to live as it is.

Reply:
ah it is not - granted legacy has all the fields